steering_vectors/train_steering_vector.py

```diff
@@ -118,14 +97,76 @@ def train_steering_vector(
             if move_to_cpu:
                 neg_act = neg_act.cpu()
             neg_activations[layer_num].append(neg_act)
+
+    return pos_activations, neg_activations
+        
+
+def aggregate_activations(
+    pos_acts: dict[int, list[Tensor]],
+    neg_acts: dict[int, list[Tensor]],
```

Comment:
nit: why change the name to `acts` instead of `activations`?

Reply:
Maybe it makes more sense to name this `pos_activations_by_layer` or `pos_acts_by_layer` to make it clear this param is a dict of layers to actications?